packages/components/src/custom-select-control-v2/types.ts

```diff
@@ -77,6 +77,10 @@ export type _CustomSelectProps = CustomSelectButtonProps & {
 	 * Accessible label for the control.
 	 */
 	label: string;
+	/**
+	 * Described by.
```

Comment:
Please use a more descriptive description of this prop. Something like `An accessible description for the control` could work.